ballerina/Ballerina.toml

```diff
@@ -5,8 +5,8 @@ name = "zoom.scheduler"
 version = "1.0.0"
 license = ["Apache-2.0"]
 authors = ["Ballerina"]
-keywords = []
-# icon = "icon.png" # TODO: update icon.png
+keywords = [] # TODO: Add keywords
+# icon = "icon.png" # TODO: Add icon
```

Comment:
Let's add the relevant keywords and icon here. Update the `build-config/resources/Ballerina.toml` file. Don't edit this file directly.

Possible keywords are `["Zoom", "Video Conference", "Scheduling"]`